plugins/nginx-vhosts/functions

```diff
@@ -22,6 +22,34 @@ fn-nginx-access-log-path() {
   fn-plugin-property-get-default "nginx" "$APP" "access-log-path" "${NGINX_LOG_ROOT}/${APP}-access.log"
 }
 
+fn-nginx-proxy-buffer-size() {
+  declare desc="get the configured proxy buffer size"
+  declare APP="$1"
+
+  fn-plugin-property-get-default "nginx" "$APP" "proxy-buffer-size" "$(fn-get-pagesize)"
+}
+
+fn-nginx-proxy-buffering() {
+  declare desc="get the configured proxy buffering"
+  declare APP="$1"
+
+  fn-plugin-property-get-default "nginx" "$APP" "proxy-buffering" "on"
+}
+
+fn-nginx-proxy-buffers() {
+  declare desc="get the configured proxy buffers"
+  declare APP="$1"
+
+  fn-plugin-property-get-default "nginx" "$APP" "proxy-buffers" "8 $(fn-get-pagesize)"
```

Comment:
Continued from https://github.com/dokku/dokku/pull/4138#discussion_r490412785:
> Should the `8` be computed?

Can you please explain a bit more why `8` should be computed? The default buffer number should be static (fixed) according to [Nginx's docs](https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffers).

Reply:
Hmmm fair, didn't read that this was `number`. I guess as a default, its fine.